lib/commands/signout.js

```diff
@@ -15,7 +15,10 @@ module.exports = async (req, res) => {
   await signout(robot, slackUser, slackWorkspace);
 
   const state = new SignedParams({
-    trigger_id: command.trigger_id,
```

Comment:
Just a general question: Do we have conventions around omitting the key vs setting it's value to `null` like: `trigger_id: null` ? If not, I would like to have your, and/or the teams perspective on it. Just to get to know how we work.

Reply:
It's a really good question, I don't think we have a very clear pattern/rule for this across the codebase. Do you have any thoughts as to what a good pattern might look like?

As extra context, `trigger_id` should only be needed now when we also pass `replaySlashCommand`. If we were using Typescript, there would probably be a nice way to express this in the code 😄